app/components/forms/wizard/basic-details-step.js

```diff
@@ -346,6 +346,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       }));
     },
 
+    salesEndDate(eventStartDate) {
+      this.get('data.event.tickets').forEach(ticket => {
+        if (ticket.get('salesEndsAt').format('MM/DD/YYYY') > eventStartDate) {
```

Comment:
why do we need to format the date for comparing. Standard moment objects should be used for any comparisons.

Reply:
The input coming from the datepicker is a string and hence the format is used. Should I convert the string to moment object?

Reply:
@Anupam-dagar Yes.